content/2_Bronze/Time_Comp.mdx

```diff
@@ -137,14 +137,14 @@ while (i < n) {
 <PySection>
 
 ```py
-for i in range(1, n+1):
-	# constant time code here
+for i in range(1, n + 1):
+	pass # constant time code here
 ```
 
 ```py
 i = 0
 while (i < n):
-	# constant time code here
+	pass # constant time code here
```

Comment:
I'm not sure if I like the use of `pass` here. Especially for the while loop which already runs fine without the `pass`, since it's another thing to search up. Have you considered putting some "dumb" placeholder code as an alternative?

Reply:
`pass`'s one purpose is to be a placeholder, isn't it?

Reply:
Isn't pass unnecessary since there's i+=1?